### PR TITLE
Update CIME to ESMCI cime5.8.13

### PR DESCRIPTION
**Pull request by jgfouca**
Update CIME to ESMCI cime5.8.13

Squash merge of jgfouca/branch-for-to-acme-2019-10-28

Features:
* Updates to workflow support in CIME.
* Add namelist for minimum wind speed for atmOcn fluxes.
* Change globally uniform SST mode from AQP11 to AQPCONST
* Add --only-job option to case.submit: run only jobname and ignore any defined workflow
* Adds a raw xml option to the query_config interface.
* Refactor how run_cmd handles strings

Bug fixes:
* Fix an issue with Lockedfile error when case.submit is used with the --no-batch flag

[NML]
[BFB]

### CODE REVIEW

File: cime/src/drivers/mct/cime_config/namelist_definition_drv.xml [hunk 1]
```diff
@@ -4749,4 +4749,17 @@
...
+  <entry id="seq_flux_atmocn_minwind">
+    <type>real</type>
+    <category>seq_flux_mct</category>
+    <group>seq_flux_mct_inparm</group>
+    <desc>
+       minimum wind speed for atmOcn flux calculations
+    </desc>
+    <values>
+      <value>0.5</value>
+      <value aqua_planet_sst_type="sst_aquap11">1.0</value>
```
Comment (whannah1) [inline cime/src/drivers/mct/cime_config/namelist_definition_drv.xml]: We need to change "sst_aquap11" to "sst_aquap_constant"

Comment (whannah1) [reply cime/src/drivers/mct/cime_config/namelist_definition_drv.xml]: I can do this, but I'm not sure how. Should I commit directly to this branch? Or to a CIME branch?

Comment (jgfouca) [reply cime/src/drivers/mct/cime_config/namelist_definition_drv.xml]: @whannah1 , thanks for looking at this. I will make the change.

Comment (jgfouca) [reply cime/src/drivers/mct/cime_config/namelist_definition_drv.xml]: @whannah1 , ok, see if you approve of my change.

File: cime/src/drivers/mct/cime_config/namelist_definition_drv.xml [hunk 2]
```diff
@@ -4749,4 +4749,18 @@
...
+    <category>seq_flux_mct</category>
+    <group>seq_flux_mct_inparm</group>
+    <desc>
+       minimum wind speed for atmOcn flux calculations
+    </desc>
+    <values>
+      <value>0.5</value>
+      <value cime_model='cesm' aqua_planet_sst_type="sst_aquap11">1.0</value>
+      <value cime_model='e3sm' aqua_planet_sst_type="sst_aquap_constant">1.0</value>
+    </values>
```
Comment (whannah1) [inline cime/src/drivers/mct/cime_config/namelist_definition_drv.xml]: This looks good for E3SM, but CESM is also going to use "sst_aquap_constant", so it should be the same for both models.

Comment (jgfouca) [reply cime/src/drivers/mct/cime_config/namelist_definition_drv.xml]: @whannah1 , done

**Comment by jgfouca** [comment]
Squash-merged to next.

**Pull request merged**